### PR TITLE
Improve comparison language

### PR DESCRIPTION
**Pull request by banesullivan**
Upon reflection, some of the language used in PyVista's documentation is not inline with the values I believe we should hold. 

Specifically, some language in PyVista's documentation comparing PyVista to VTK sheds a negative light on VTK, failing to acknowledge the hard work of those who developed VTK and the value VTK has proven to hold in its 20+ year history.

I believe it is of the utmost importance to consider the fact that PyVista stands on shoulders of giants, namely VTK and NumPy, and that we should acknowledge that fact while being respectful to the broader communities behind those software.

This pull request specifically changes a few concerning language choices:

- "VTK for humans" -> "Pythonic VTK": while the phrasing was originally contributed by a member of our community, inline with many other tools providing a simplified API to a powerful library (e.g., [requests](https://2.python-requests.org/en/master/)'s wrapping of `urllib3` and "HTTP for Humans™" tagline), and capturing the beauty of PyVista's simplified API, it demonstrates a lack of respect for the humans who created VTK and their hard work building the backbone of PyVista.
- I also noticed a few places where the documentation provides a comparison of PyVista and VTK that portrays VTK in a negative light. While the intent was good (to demonstrate the API that the PyVista community has come to love), it is not fair or respectful to shed a negative light on VTK. The use cases for PyVista and VTK Python are different; both have their advantages and drawbacks and we should be sure to honor that.

Additionally, I added some notes to our Code of Conduct to encourage the use of respectful language moving forward.

I suspect I most likely did not capture all occurrences of non-flattering language and would like to ask @pyvista/developers to please follow up in other Pull Requests to improve our language across the documentation

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ diverse, inclusive, and healthy community.
 Examples of behavior that contributes to a positive environment for our
 community include:
 
-* Demonstrating empathy and kindness toward other people
```
Comment (tkoyama010) [inline CODE_OF_CONDUCT.md:21]: CODE_OF_CONDUCT.md is taken from Contributor Covenant, version 2.1. It is not good for maintenance when upgrading Contributor Covenant to change the wording of the quotation, so it is preferable to leave it as is.

**Review by tkoyama010 — COMMENTED** [review]
@banesullivan Thank you for pointing this out. I've always felt that PyVista should be more respectful of VTK. I have one point comment for this changes.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (banesullivan) [reply CODE_OF_CONDUCT.md:21]: @tkoyama010, do you have a suggestion on where we could put this? Perhaps in the CONTRIBUTING.md guide?

Comment (tkoyama010) [reply CODE_OF_CONDUCT.md:21]: @banesullivan Agree. CONTRIBUTING.md is a good place to add this.

File: doc/getting-started/authors.rst [hunk 2]
```diff
@@ -2,12 +2,20 @@ Authors & Citation
...
 .. _Contributors Graph: https://github.com/pyvista/pyvista/graphs/contributors/
 
 
+.. note::
+
+  PyVista stands on the shoulders of giants, namely the Visualization Toolkit
+  (VTK) and NumPy. We owe so many thanks to the companies and communities that
+  have worked diligently to build the software stack that makes PyVista
+  possible.
+
```
Comment (akaszynski) [inline doc/getting-started/authors.rst:17]: Thanks for adding this.

**Review by akaszynski — APPROVED** [review]
Thanks for adding this.

I have always been amazed at VTK's speed and usability, even without wrapping it with PyVista. The original intent of this library was to increase the adoption of VTK in the Python community by making it seem more "Pythonic", while retaining (most) of the performance of the VTK library, and not to replace it (which would be next to impossible). I agree wholeheartedly that we do indeed "stand on the shoulders of giants."

I'll be on the lookout for additional sections that need tweaking.

**Review by tkoyama010 — APPROVED** [review]

**Pull request merged**